### ISSUE TITLE
Support for new ExecuteProcessWithReturn

### ISSUE THREAD

**Issue by rclapp** [cubewise-code/TM1py#65]
Has the code been updated to use the new TI execution standard? Based on conversations with IBM moving to the new function will correct issues that keep minor errors from being committed

**Comment by cubewise-tryan** [comment]
Hi @rclapp,

It is coming, beware there is a bug in 11.3 meaning it can't be used unless you have installed at least IF1.

**Comment by MariusWirtz** [comment]
`execute_with_return function` was merged with #68 

function returns three values:
1: success (True or Talse)
2: execution-state (String): E.g. 'CompletedSuccessfully', 'CompletedWithMessages', 'Aborted', ...
3: None or file name of ErrorLogFile (String)

you can use third value in the `get_error_log_file_content` function to access the actual content of the file:
```

import uuid

from TM1py import TM1Service, Process

with TM1Service(address='10.77.19.10', port=12354, user='admin', password='apple', ssl=True, logging=False) as tm1:
    process = Process(name=str(uuid.uuid4()))
    process.epilog_procedure = "ItemReject('Not Relevant');"

    if not tm1.processes.exists(process.name):
        tm1.processes.create(process)

    success, status, error_log_file = tm1.processes.execute_with_return(process_name=process.name)

    content = tm1.processes.get_error_log_file_content(file_name=error_log_file)
    print(content)

    tm1.processes.delete(process.name)

```

@rclapp 
does that work for you? Feel free to open a MR if you need something different!
